src/Microsoft.AspNetCore.Blazor.Server/BlazorConfig.cs

```diff
@@ -29,6 +29,9 @@ private BlazorConfig(string assemblyPath)
             var configLines = File.ReadLines(configFilePath).ToList();
             SourceMSBuildPath = configLines[0];
 
+            if (string.IsNullOrEmpty(SourceMSBuildPath) || SourceMSBuildPath == ".")
```

Comment:
Is there any scenario where `SourceMSBuildPath` is null or empty? I'm not aware that there is. If I'm correct, could you revert that part of the `if` clause?

Reply:
Actualy no, i have only seen "." instead, maybe i have overdone this. We can assume that "." case is the only alternate scenario possible actually.

I will modify this part, thanks !